src/main/resources/application.conf

```diff
@@ -6,7 +6,7 @@ centaur {
   # The maximal length of a workflow, intended as a sanity check and not actually a test in and of itself
   maxWorkflowLength: 2 hours
   # Cromwell's metadata is eventually consistent. Set a timeout such that we expect it to have eventually consisted
-  metadataConsistencyTimeout: 10 seconds
+  metadataConsistencyTimeout: 5 minutes
```

Comment:
Yes I agree. FYI I created a cromwell branch pointing to this centaur branch and haven't seen a `workflowName` error yet. I relaunched it 3 times.
While agree this is not an acceptable solution it's rapidly becoming an annoying problem as right now it's getting hard to get a green PR. For some reason a significant number of workflows start failing because they can't upload the auth file as well..